### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

This cleans up a few things with regard to the CI config

* point to master branch of industrial_ci as clang-tidy feature was merged
* remove unnecessary quotes from yaml config
* rename ci files to have correct yaml file name extension
* fixup readme badges to be more descriptive and point to more useful links

### CODE REVIEW

**Review by vatanaksoytezer — APPROVED** [review]
This cleans up the file a little, which looks great. Looks good to me!

File: README.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ The MoveIt Motion Planning Framework for **ROS 2**. For ROS 1, see [MoveIt 1](ht
 
 ## Continuous Integration Status
 
-[![Format](https://github.com/ros-planning/moveit2/actions/workflows/format.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/format.yml?branch=main) [![BuildAndTest](https://github.com/ros-planning/moveit2/actions/workflows/industrial_ci_action.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/industrial_ci_action.yml?branch=main) [![codecov](https://codecov.io/gh/ros-planning/moveit2/branch/main/graph/badge.svg?token=W7uHKcY0ly)](https://codecov.io/gh/ros-planning/moveit2)
+
+[![Formatting (pre-commit)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml?query=branch%3Amain)
```
Comment (henningkayser) [inline README.md:10]: I just randomly tried this link and got "No results matched your search". Somehow all workflows are listed twice on that page and when selecting the other one (the one not pointed to by default), the search "branch:main" shows all the results.  We could use "https://github.com/ros-planning/moveit2/actions?query=branch%3Amain+workflow%3AFormat" to fix this.

Comment (tylerjw) [reply README.md:10]: The reason you get no results is that I renamed the file from `format.yml` to `format.yaml` in this PR and that isn't on main yet.

Comment (henningkayser) [reply README.md:10]: :man_facepalming:

**Review by henningkayser — APPROVED** [review]

**Pull request merged**